doc/usage/extensions/napoleon.rst

```diff
@@ -511,3 +539,9 @@ sure that "sphinx.ext.napoleon" is enabled in `conf.py`::
        :type arg2: :term:`dict-like <mapping>`
 
    .. versionadded:: 3.2
+
+.. confval:: napoleon_google_attr_annotations
```

Comment:
Now I'm thinking to apply this to numpy also. There is no reason to limit this feature to google style only.

Reply:
I started looking at that, I'm not too sure how to do it yet. Is `sphinx.ext.napoleon.docstring.NumpyDocstring::_consume_field()` the right place ?